src/Internal/FileTask.php

```diff
@@ -54,8 +54,7 @@ public function run(Channel $channel, Cancellation $cancellation): mixed
             if ("fopen" === $this->operation) {
                 $file = $driver->openFile(...$this->args);
 
-                $size = $driver->getStatus($file->getPath())["size"]
-                    ?? throw new FilesystemException("Could not determine file size");
+                $size = $driver->getStatus($file->getPath())["size"] ?? 0;
```

Comment:
Why did this change?

Reply:
Oops, leftover when I was trying to figure out what Windows was doing. Reverted.